### DISCUSSION TITLE
What's the best way to share osu lazer data from one device to another?

### DISCUSSION THREAD

**Discussion by zMienai** [ppy/osu#26737] [Q&A]
Hey, i want to transfer my local osu! lazer data from my pc to my tablet but i don't know what's the safest and quickest way to do so.

In case there's no internal way to do this in osu!, i feel like there could be like a cloud saving system, something similar to geometry dash.

**Comment by peppy** [comment]
copy the folder

**Reply by zMienai** [reply]
That's what i was doing, and is it really the most efficient way to do this? A cloud saving system could be good in case people want to transfer their data to different devices

**Reply by mixalisnew** [reply]
Would be expenice is my guess

**Reply by marioman63** [reply]
what would you expect it to include in the cloud save? songs are a definite no go, some of us have hundreds of GB (my folder is almost up to 700 GB, for reference). If you don't include songs, then maybe a table of what's installed. But what if said song isn't available for download? Well now you have to do it manually anyways. Some people may not have speedy internet or limited data as well, so they might just opt for a local manual option too. I'd imagine it's not worth it for what would be the most reasonable data to upload.

**Reply by zMienai** [reply]
Lazer has a function where if you download a play and the map is missing, then the map will automatically download, so the songs don't matter.
